Library/HelpTypeTests.swift

```diff
@@ -0,0 +1,14 @@
+import XCTest
+import Library
+
+final class HelpTypeTests: XCTestCase {
+  func testAccessibilityTraits() {
+    XCTAssertEqual(HelpType.helpCenter.accessibilityTraits, UIAccessibilityTraits.link)
```

Comment:
I've provided tests for all the cases even though we don't currently show `howItWorks` and `trust` which I believe show a web view as well.

Reply:
Nice, thanks for adding!